src/ripple/app/misc/Validations.cpp

```diff
@@ -93,6 +93,10 @@ class ValidationsImp : public Validations
 
         auto pubKey = app_.validators ().getTrustedKey (signer);
 
+        // Do not process or forward partial validations.
```

Comment:
Good question.  I fixed to properly forward partial validations.  At this time, we aren't actually using them to do anything special aside from including them in the partial validation stream, but you are correct that they should be forwarded.